<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QWidget" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1184</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">#Editor {
      border-image: url(:/images/fondo.jpg) 0 0 0 0 stretch stretch;
}

#Editor QLabel {
    border: 1px solid #FFA500;
    padding: 0px;
    background-color: #000000;
    qproperty-alignment: 'AlignCenter';
    min-width: 32px;  /* Cambiado a 32px */
    min-height: 32px; /* Cambiado a 32px */
    margin: 0px;
}

#Editor QLabel[scaledContents=&quot;true&quot;] {
    qproperty-scaledContents: false;  /* Cambiado a false para evitar escalado autom√°tico */
}

#Editor QLabel:hover {
    border: 1px solid #FFD700;
    background-color: rgba(255, 165, 0, 0.1);
}
#Editor QScrollArea {
border: 2px solid #FFA500;
background-color: #000000;
margin: 0px;
padding: 0px;
}

#Editor QScrollArea QWidget {
background-color: #000000;
}

#Editor QScrollArea QScrollBar:vertical {
width: 12px;
background: #1a1a1a;
border: none;
margin: 0px;
}

#Editor QScrollArea QScrollBar::handle:vertical {
background: #FFA500;
border-radius: 6px;
min-height: 20px;
}

#Editor QScrollArea QScrollBar::handle:vertical:hover {
background: #FFD700;
}

#Editor QScrollArea QScrollBar::add-line:vertical,
#Editor QScrollArea QScrollBar::sub-line:vertical {
height: 0px;
}

#Editor QScrollArea QScrollBar:horizontal {
height: 12px;
background: #1a1a1a;
border: none;
margin: 0px;
}

#Editor QScrollArea QScrollBar::handle:horizontal {
background: #FFA500;
border-radius: 6px;
min-width: 20px;
}

#Editor QScrollArea QScrollBar::handle:horizontal:hover {
background: #FFD700;
}

#Editor QGridLayout {
margin: 0px;
padding: 0px;
spacing: 0px;
} 
QPushButton {
    background-color: #2C1810;  
    border: 4px solid #FFB626;  
    border-style: solid;
    border-bottom-width: 8px;   
    color: #FFD700;            
    font-family: &quot;Press Start 2P&quot;, &quot;VT323&quot;, &quot;Consolas&quot;; 
    font-size: 28px;
    text-transform: uppercase;
    padding: 15px 30px;
    margin: 10px;
    border-radius: 0px;        
    font-weight: bold;
}
QPushButton:hover {
    background-color: #3C2820;  
    border-bottom-width: 10px;  
}
QPushButton:pressed {
    background-color: #1C0800;  
    border-bottom-width: 4px;   
}
QPushButton:disabled {
    background-color: #1A1A1A;
    border-color: #666666;
    color: #888888;
}
QLabel {
    background-color: #2C1810;  
    border: 4px solid #FFB626;  
    border-style: solid;
    border-bottom-width: 8px;   
    color: #FFD700;            
    font-family: &quot;Press Start 2P&quot;, &quot;VT323&quot;, &quot;Consolas&quot;; 
    font-size: 28px;
    text-transform: uppercase;
    padding: 15px 30px;
    margin: 10px;
    border-radius: 0px;        
}
QLineEdit, QComboBox {
      border: 4px solid #df9f1f; 
      color: #df9f1f; 
      font-family: &quot;Arial Black&quot;;
      font-size: 35px;
      font-weight: bold;
      text-transform: uppercase;
      padding: 15px 30px;
      margin: 10px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="0" column="1">
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="mapWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1098</width>
        <height>465</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="mapLayout">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tilesScroll">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="tilesLayout">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QPushButton" name="ExitButton">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="LoadButton">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SaveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EraseButton">
       <property name="text">
        <string>Erase</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EraiserButton">
       <property name="text">
        <string>Eraser</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="backgroundBox">
       <item>
        <property name="text">
         <string>Forest</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lava</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nieve</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
